src/config/config.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+
+module.exports = {
+  "development": {
+    "username": process.env.DB_USERNAME,
+    "password": process.env.DB_PASSWORD,
+    "database": process.env.DB_NAME,
+    "host": process.env.DB_HOST,
+    "dialect": "postgres",
+    "operatorsAliases": false
+  },
+  "test": {
+    "username": process.env.DB_TEST_USER,
+    "password": process.env.DB_TEST_PASS,
```

Comment:
you only need the `DB_TEST_NAME` to be different. the rest can use the same config.

Reply:
Okay